<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1758</width>
    <height>1197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,6">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Layers">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>389</width>
         <height>1030</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Layers</string>
       </attribute>
       <layout class="QGridLayout" name="LayersGridLayout">
        <item row="0" column="0">
         <widget class="QScrollArea" name="LayersScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="LayersScrollAreaContent">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>363</width>
             <height>1004</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PageNav">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>246</width>
         <height>1030</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page Navigator</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QScrollArea" name="PageNavScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="PageNavScrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>220</width>
             <height>1004</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="PagesGridLayout">
            <item row="0" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="SVGScrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1333</width>
         <height>1096</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QGraphicsView" name="svgView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1758</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="actionsave_as"/>
    <addaction name="separator"/>
    <addaction name="actionclose"/>
   </widget>
   <widget class="QMenu" name="menuedit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actioncopy"/>
    <addaction name="actioncut"/>
    <addaction name="actionpaste"/>
    <addaction name="separator"/>
    <addaction name="actionundo"/>
    <addaction name="actionredo"/>
   </widget>
   <widget class="QMenu" name="menuimport">
    <property name="title">
     <string>&amp;Import</string>
    </property>
    <addaction name="actionimage"/>
    <addaction name="actionIIIF"/>
    <addaction name="separator"/>
    <addaction name="actiondemo_object"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuedit"/>
   <addaction name="menuimport"/>
   <addaction name="menuExport"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionPath_Cutter"/>
   <addaction name="actionMove"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionPath_editor"/>
   <addaction name="separator"/>
   <addaction name="actionGroup"/>
   <addaction name="actionUngroup"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Line_Path"/>
   <addaction name="actionAdd_Curve_Path"/>
   <addaction name="separator"/>
   <addaction name="actionSave_selected_object_as_demo_object"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
  </widget>
  <action name="actionopen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="text">
    <string>copy</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="text">
    <string>cut</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="text">
    <string>paste</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="text">
    <string>undo</string>
   </property>
  </action>
  <action name="actionredo">
   <property name="text">
    <string>redo</string>
   </property>
  </action>
  <action name="actionsave_as">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
  <action name="actionimage">
   <property name="text">
    <string>image</string>
   </property>
  </action>
  <action name="actionIIIF">
   <property name="text">
    <string>IIIF</string>
   </property>
  </action>
  <action name="actiondemo_object">
   <property name="text">
    <string>demo object</string>
   </property>
  </action>
  <action name="actionsave_as_demo_object">
   <property name="text">
    <string>save as demo object</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select Tpp;</string>
   </property>
  </action>
  <action name="actionPath_Cutter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Path Cutter</string>
   </property>
   <property name="toolTip">
    <string>Used to cut a vector path</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Move selected object</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete Object</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Meta+Z, Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo action</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+Z, Meta+Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionPath_editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Path editor</string>
   </property>
   <property name="toolTip">
    <string>Switch to Path Editing Mode</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="text">
    <string>Group</string>
   </property>
   <property name="toolTip">
    <string>Group Selected Paths to new Object</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G, Meta+G</string>
   </property>
  </action>
  <action name="actionUngroup">
   <property name="text">
    <string>Ungroup</string>
   </property>
   <property name="toolTip">
    <string>Ungroup selected object to individual Paths</string>
   </property>
  </action>
  <action name="actionAdd_Line_Path">
   <property name="text">
    <string>Add Line Path</string>
   </property>
   <property name="toolTip">
    <string>Add a new line vector path</string>
   </property>
  </action>
  <action name="actionAdd_Curve_Path">
   <property name="text">
    <string>Add Curve Path</string>
   </property>
   <property name="toolTip">
    <string>Add new Cubic Bezier Curve vector path</string>
   </property>
  </action>
  <action name="actionDelete_2">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSave_selected_object_as_demo_object">
   <property name="text">
    <string>Save selected object as demo object</string>
   </property>
   <property name="toolTip">
    <string>Save selected object as demo object</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>SVGScrollArea</tabstop>
 </tabstops>
 <resources>
  <include location="resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
